scripts/mbedtls_framework/code_wrapper/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+### NOTE: The code on this file is importing modules from
+### The main mbedtls repository. Those files will eventually
+### be migrated here.
+
+import os, sys, pkgutil
+
+_p = ["tests", "scripts", "code_wrapper"]
+
+for _ in range(5):
+    try:
+        sys.path.append(os.path.join(*_p))
+        import psa_test_wrapper, psa_buffer, psa_wrapper
+    except ModuleNotFoundError:
+        _p = ["../"] + _p
+        continue
+
```

Comment:
I don't know why, but on my PC I have 2 empty lines at the end of this file, whereas here on github I see only 1. In case there are really 2 lines I would ask to remove 1 whereas if it's already correct, please ignore this comment.